src/app/FakeLib/UnitTest/NUnit/Common.fs

```diff
@@ -34,29 +34,53 @@ type NUnitDomainModel =
         | DefaultDomainModel -> ""
         | SingleDomainModel -> "Single"
         | MultipleDomainModel -> "Multiple"
-/// Parameter type for NUnit.
+
+/// The [NUnit](http://www.nunit.org/) Console Parameters type.
+/// FAKE will use [NUnitDefaults](fake-nunitcommon.html) for values not provided.
+///
+/// For reference, see: [NUnit-Console Command Line Options](http://www.nunit.org/index.php?p=consoleCommandLine&r=2.6.4)
 type NUnitParams = 
```

Comment:
Yes, it looks wrong. I will change it to the same style as your comments to xUnit then.